### PR TITLE
Fall 2017

### PR DESCRIPTION
**Pull request by oerms**
Patched integration of URS.

### CODE REVIEW

**Comment by oerms** [comment]
Who will be reviewing this?

**Comment by Azurency** [comment]
I am reviewing it

File: Integrations/URS/reportscreen.lua [hunk 1]
```diff
@@ -1334,7 +1313,7 @@ function ViewYieldsPage()
           if kBuilding.Name == Locale.Lookup( GameInfo.Districts[i].Name ) and GameInfo.Districts[i].Maintenance > 0 then
             local pBuildingInstance:table = {};
             ContextPtr:BuildInstanceForControl( "BuildingExpensesEntryInstance", pBuildingInstance, instance.ContentStack );
-            pBuildingInstance.CityName:SetText( Locale.Lookup(cityName) );
+            TruncateStringWithTooltip(pDistrictInstance.CityName, 224, Locale.Lookup(cityName));
```
Comment (Azurency) [inline Integrations/URS/reportscreen.lua]: Should be `pBuildingInstance.CityName`

File: Integrations/URS/reportscreen.xml [hunk 2]
```diff
@@ -168,13 +168,13 @@
...
   <Instance						Name="GroupInstance">
-    <Container				ID="Top"																					Size="1003,10"	AutoSize="V">
+    <Container				ID="Top"																					Size="1003,auto">
       <GridButton			ID="RowHeaderButton"								Offset="0,0"	Size="parent,30"		Style="RowButton"		String="$RowHeaderButton" >
         <Label ID="RowHeaderLabel" Anchor="R,C" Offset="50,0" Style="RowButton" String="Ends in ## turns (###)" Hidden="1" />
       </GridButton>
       <Image					ID="RowExpandCheck"		Anchor="R,T"	Offset="5,5"	Size="22,22"				Texture="Controls_ExpandButton" />
-      <ScrollPanel		ID="CollapseScroll"									Offset="0,31"	Size="parent,20"		StackGrowth="Bottom" FullClip="1" >
-        <SlideAnim		ID="CollapseAnim"			Start="0,0"	EndOffset="0,0" Speed="4" Cycle="Once" Stopped="1">
+      <ScrollPanel		ID="CollapseScroll"									Offset="0,31"	Size="parent,auto"	StackGrowth="Bottom" FullClip="1" >
```
Comment (Azurency) [inline Integrations/URS/reportscreen.xml]: `parent,parent` instead of `parent,auto`, otherwise there's a bug when you collapse the rows.

**Comment by Azurency** [comment]
It's merged 👍

**Comment by ucolaf** [comment]
meet another civilization ui hiding

**Comment by oerms** [comment]
@ucolaf I am afraid I do not understand. What do you mean?

**Comment by ucolaf** [comment]
when i found a new civilisation, then my complete UI is lost, i have a map with no UI, workaround, "ESC", SAVE,LOAD

**Comment by oerms** [comment]
@ucolaf 
Official testing of the fall-2017 patch has not started yet. We are waiting for the integrations being fully merged. If I understand @Azurency correctly, he wants to do this tonight. If the bug still persist after that, please report the bug here: https://github.com/Azurency/CQUI_Community-Edition/issues/152
Thank you!

**Pull request merged**